contributing.md

```diff
@@ -11,10 +11,12 @@
 5. Fork this repository! Once you have a copy of this repo on your own account, clone this repo to your computer by typing in something like:
   `git clone https://github.com/codebuddiesdotorg/cb-v2-scratch.git`
 (Replace the URL with your own repository URL path.)
-6. Type `meteor npm install`, and then `meteor --settings settings-development.json` in your terminal to start up the app in your browser (`localhost:3000`). (`npm run meteor:dev` can also run the app, but will likely [use up your CPU](https://github.com/meteor/meteor/issues/4314).)
-7. Look at some of the [open issues](https://github.com/codebuddiesdotorg/cb-v2-scratch/issues) and identify one that sparks your interest. If you want to work on the issue, leave a comment on it saying that you're working on it!
-8. If you have any questions about the issue you're looking at, you can leave a comment in there, or ask in the [#codebuddies-meta Slack channel](https://codebuddiesmeet.slack.com/messages/codebuddies-meta). Read below for more instructions about how to work with branches.
-9. Type `git branch -a` to see a list of all the branches besides `master`, the default branch you're in. Note that if you want to switch to a branch returned in the last, you would type `git checkout BRANCHNAME`. You can read more about how to create a new branch to work on an issue below.
+6. Setup this repository as an upstream branch using:
+`git remote add upstream https://github.com/codebuddiesdotorg/cb-v2-scratch/.git`. <br/>Now, whenever you want to sync with the owner repository. Do the following: (i) `git fetch upstream` (ii) `git checkout master` (iii) `git merge upstream/master`.
```

Comment:
You have a typo here for the repo's url. It should be the following:

https://github.com/codebuddiesdotorg/cb-v2-scratch.git

Also, for (i), (ii), and (iii), could you write those out as a either an ordered list or unordered list so that each command is on its own line? This makes it easier for the user to just copy and paste each command line command.